core/src/main/kotlin/links/DRI.kt

```diff
@@ -1,5 +1,9 @@
 package org.jetbrains.dokka.links
 
+import org.jetbrains.dokka.model.ClassKind
+import org.jetbrains.dokka.model.Documentable
+import org.jetbrains.dokka.model.WithSupertypes
```

Comment:
You don't need those imports if I am correct

Reply:
Sure, removed